README.md

```diff
@@ -1,7 +1,7 @@
 # Super Simple Time Tracker
 Multi-purpose time trackers for your notes!
 
-![A screenshot of the plugin in action, where you can see an active time tracker for a project](https://raw.githubusercontent.com/Ellpeck/ObsidianSimpleTimeTracker/master/screenshot.png)
+![A screenshot of the plugin in action, where you can see an active time tracker for a project](screenshot.jpg)
```

Comment:
This needs to be an absolute url as otherwise it doesn't get displayed properly in the plugin store thingy in Obsidian itself

Reply:
Ufff, you're right! I forgot about the obsidian store :D It's fixed :)